app/main/views/direct_award.py

```diff
@@ -141,6 +143,45 @@ def get_project(project_external_id):
     return single_result_response("project", project, serialize_kwargs={"with_users": True}), 200
 
 
+@main.route('/direct-award/projects/<int:project_external_id>', methods=['PATCH'])
+def update_project(project_external_id):
+    uniform_now = datetime.datetime.utcnow()
+
+    updater_json = validate_and_return_updater_request()
+
+    json_payload = get_json_from_request()
+    json_has_required_keys(json_payload, ["project"])
+    project_json = json_payload["project"]
+
+    validate_direct_award_project_json_or_400(project_json)
+
+    # fetch and lock row
+    project = db.session.query(DirectAwardProject).filter_by(
+        external_id=project_external_id,
+    ).with_for_update().first()
+
+    if not project:
+        abort(404, f"Project {project_external_id} not found")
+
+    # TODO: project.update_from_json(project_json)
+
+    if project_json.get('stillAssessing'):
```

Comment:
Here's a thought - should we be allowing `stillAssessing` to be set if we've already got an `outcome` for this project?

Reply:
Will do

Reply:
Should we then make it so that `stillAssessingAt` is set to `None` when an outcome is created?

Reply:
I would say no, we still want to retain that data for our records. If you think it feels weird it perhaps would make sense to think of the `stillAssessingAt` field as "was still assessing at <this time>" rather than a field flagging that the project _is_ currently in a particular state.

Reply:
Done